content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Google Cloud Configuration Guide for Cloud SIEM
+kind: documentation
+further_reading:
+- link: "/security/default_rules/#cat-cloud-siem-log-detection"
+  tag: "Documentation"
+  text: "Explore Cloud SIEM default detection rules"
+- link: "/security/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Security Signals Explorer"
+- link: "/security/cloud_siem/log_detection_rules/"
+  tag: "Documentation"
+  text: "Create new detection rules"
+- link: "https://www.datadoghq.com/blog/visualize-google-cloud-activity-cloud-siem-investigator/"
+  tag: "Blog"
+  text: "Visualize activity in your Google Cloud environment with Datadog Cloud SIEM Investigator"
+---
+
+## Overview
+
+[Datadog Cloud SIEM][1] applies detection rules to all processed logs in Datadog to detect threats, like a targeted attack, a threat intel listed IP communicating with your systems, or an insecure resource modification. The threats are surfaced as Security Signals in the Security Signals Explorer for triaging.
+
+This guide walks you through the following steps so that you can start detecting threats with your Google Cloud audit logs:
+
+1. [Enable Data Access audit logs](#enable-data-access-audit-logs)
+1. [Create a new Google Cloud publish/subscription (pub/sub) system](#create-a-new-google-cloud-publishsubscription-pubsub-system)
+1. [Configure the pub/sub to send logs to Datadog](#configure-the-pubsub-to-send-logs-to-datadog)
+1. [Export logs from Google Cloud Logging to the pub/sub](#export-logs-from-google-cloud-logging-to-the-pubsub)
```

Comment:
should we add the last step here as "[Use Cloud SIEM to triage Security Signals]" similar to AWS guide?

Reply:
Good catch, added it!